### PR TITLE
ES-677 Fix Year is not displayed in account last updated password

### PR DESCRIPTION
**Pull request by seta-tuha**
Link to ticket: https://steel-ventures.atlassian.net/browse/ES-677

### CODE REVIEW

File: packages/veritone-widgets/package.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
     "redux-thunk": "^2.3.0",
     "shaka-player": "^2.4.1",
     "veritone-oauth-helpers": "^2.1.0",
-    "video-react": "~0.13.3"
+    "video-react": "~0.13.3",
+    "reselect": "^4.0.0"
```
Comment (MmtBkn) [inline packages/veritone-widgets/package.json:29]: Why did we add reselect, remove it if not used.

File: packages/veritone-widgets/src/widgets/UserProfile/PasswordField/index.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const PasswordField = ({ lastUpdated, onEdit }) => {
     <div className={classes.container}>
       <RaisedTextField
         label="Password"
-        value={`Last updated ${format(new Date(lastUpdated), 'MMMM dd, yyyy')}`}
+        value={`Last updated ${format(new Date(lastUpdated), 'MMMM DD, YYYY')}`}
```
Comment (MmtBkn) [inline packages/veritone-widgets/src/widgets/UserProfile/PasswordField/index.js:18]: Problem is not the format, this value is being null. Please check app repos to see why wwe are getting this value null. We also like to show different message if password is never been updated. Something like;  

```suggestion
        value={lastUpdated ? `Last updated ${format(new Date(lastUpdated), 'MMMM DD, YYYY')}` : 'Never Updated'}
```

File: packages/veritone-widgets/package.json (hunk 1 above)
Comment (seta-tuha) [reply packages/veritone-widgets/package.json:29]: I saw it at https://github.com/veritone/veritone-sdk/blob/master/packages/veritone-widgets/src/redux/modules/folder/selector.js#L1

File: packages/veritone-widgets/src/widgets/UserProfile/PasswordField/index.js (hunk 2 above)
Comment (seta-tuha) [reply packages/veritone-widgets/src/widgets/UserProfile/PasswordField/index.js]: I logged value of `lastUpdated`, it's not null.

**Comment by MmtBkn** [comment]
@seta-tuha is this released?

**Pull request merged**